service/gcs/bridge/bridge.go

```diff
@@ -41,6 +41,9 @@ var capabilities = prot.GcsCapabilities{
 		},
 	},
 	RuntimeOsType: prot.OsTypeLinux,
+	GuestDefinedCapabilities: prot.GcsGuestCapabilities{
+		SignalProcessSupported: true,
```

Comment:
Are we going to support NamespaceAddRequestSupported at a later date?

Reply:
Yes we dont support it yet.

Reply:
SGTM